pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/model/conditions/Messages.properties

```diff
@@ -28,4 +28,6 @@ Changed.Description=Run if the current build's status is different than the prev
 Failure.Description=Run if the build status is "Failure"
 NotBuilt.Description=Run if the build status is "Not Built"
 Success.Description=Run if the build status is "Success" or hasn't been set yet
-Unstable.Description=Run if the build status is "Unstable"
\ No newline at end of file
+Unstable.Description=Run if the build status is "Unstable"
+Fixed.Description=Run if the current build is "Success" and the previous build is not
+Regression.Description=Run if the current build's status is worse than the previous build's status
```

Comment:
Shouldn't we double the single quotes as we normally should in `.properties`?  Not sure where those messages are exposed to users to check how the line 30 above is rendered already in the existing releases.

Reply:
Good catch.